executor/ddl.go

```diff
@@ -486,7 +485,7 @@ func (e *DDLExec) getRecoverTableByTableName(tableName *ast.TableName) (*model.J
 		// Check GC safe point for getting snapshot infoSchema.
 		err = gcutil.ValidateSnapshotWithGCSafePoint(job.StartTS, gcSafePoint)
 		if err != nil {
-			return nil, nil, err
+			return nil, nil, errors.Errorf("can't not find drop/truncate table '%s' in within GC safe point %s", tableName.Name.O, model.TSConvert2Time(gcSafePoint).String())
```

Comment:
Is this what you mean?
```suggestion
			return nil, nil, errors.Errorf("can't find drop/truncate table '%s' in GC safe point %s", tableName.Name.O, model.TSConvert2Time(gcSafePoint).String())
```

Reply:
yes, great catch.